RedHat-UX/red-hat-design-system
# chore: update deps

## What I did

1. update pfe deps, esbuild, and various patch versions of deps


## Testing Instructions

1. check out DPs, demos, and dev server. ideally run past @zhawkins for a spandx demo or some such

## Notes to Reviewers
